e2e/browser/test/README.md

```diff
@@ -1,4 +1,4 @@
-# End-to-end tests for solid-client-notifications in the browser
+# End-to-end tests for solid-client in the browser
```

Comment:
This change looks incorrect?

Reply:
Indeed f6722f74d955263a111480c64bff3bd77eb501e0